### ISSUE TITLE
Project 1(Bokeh gallery) Outreachy contributions-Joy Akinyi

### ISSUE THREAD

**Issue by Joyakis** [bokeh/outreach-programs#33]
@bryevdv @pavithraes 
My [github gist](https://gist.github.com/Joyakis/016ce91cafdb248e3fbbae7ec5ce065d) has the codes i used to update bokeh gallery plots . I also set up my local environment as seen in the attached file.Additionally,I will also continue to update the plots and give more solutions in regards to this project.The first plot is as it is in the gallery while the second plot is my updated plot 
  
![Local environment](https://user-images.githubusercontent.com/99389632/226188841-0669732f-f69f-4afd-8bab-85794415920f.png)

![Groupby plot as seen in the gallery](https://user-images.githubusercontent.com/99389632/226188882-2a0198bb-d8a4-4d52-b693-d1246b64739d.png)

![My Updated plot](https://user-images.githubusercontent.com/99389632/226188908-e2410b15-f933-4f66-aedf-d5134d9e1490.png)

**Comment by bryevdv** [comment]
Hi Joy, thanks for the update, we can definitely count this work towards the microtask completion. However, this particular example was already updated by another participant in a PR last week https://github.com/bokeh/bokeh/pull/12953 so if you intend to make a Pull Request against the main repo, it would be good to select another example that has not already been recently changed. 

A couple of comments:

* While cividis is definitely an improvement over the previous colors, one thing that is also consider with colormaps is whether the underlying data is discrete/categorical or continuous. The cividis colormap as I have seen it used is usually employed with continuous data (e.g. values like temperature or pressure, etc) whereas the data here is a few discrete categories. In that case it is good look for palettes that are also designed with the categorgical use-case specifically in mind. 

* I am not sure about the label angle adjustment. Just to me personally, having the rotated labels means I have to rotate my neck less in order to read them, which seems easier. But it would be interesting to know if anyone has done any actual usability studies around these kinds of questions for visualizations, if you know of or some across any references please pass them on, they could be very useful for style guidelnes in the future.

**Comment by Joyakis** [comment]
Hello @bryevdv  Thank you for the much needed feedback! I am definitely looking forward to updating more plots. Well for the xtick labels I was thinking maybe for space efficiency it would be convenient to have them at 90 degrees since they'll take up less horizontal space on the plot.

**Comment by bryevdv** [comment]
>  for the xtick labels I was thinking maybe for space efficiency it would be convenient to have them at 90 degrees since they'll take up less horizontal space on the plot.

This is definitely a valid thing to consider for sure! Another idea might be to simply swap the axes, so that the labels can just be horizontal and very easy to read. But then it would be a very "tall" plot and also I think (this is speculation on my part) the most people have an easier time judging differences in height of vertical bars, than they would judging length of horizontal bars. But depending on the chart it still might  make sense to swap axe to have horizontal axis labels. There are always trade-offs to consider!

**Comment by Joyakis** [comment]
@bryevdv @pavithraes Yeah i was having the same thoughts too....they'd look "tall" and most people are familiar with the bars being plotted vertically not horizontally,thats why i opted for changing the angles of the the xticks.I have also updated another plot,You can find the link to the gist [here](https://gist.github.com/Joyakis/81950576a5c5643eefa8af893ba289bf).Attached is the updated plot. Kindly Advice on whether i should request a PR 
![nested_update](https://user-images.githubusercontent.com/99389632/226279870-c5930e1e-ba3d-4643-b412-2160400b60dd.png)
